packages/cozy-harvest-lib/test/components/TriggerManager.spec.js

```diff
@@ -118,7 +126,9 @@ describe('TriggerManager', () => {
 
       it('should show the new account form', async () => {
         const { findByLabelText, findByTitle } = render(
-          <TriggerManager {...props} />
+          <I18n lang="en" dictRequire={() => enLocale}>
```

Comment:
Nit: we could use a setup function to avoid repeating the I18n wrapper :)

Reply:
Yes, good idea, thanks.